MaterialComponents.podspec

```diff
@@ -37,6 +37,16 @@ Pod::Spec.new do |mdc|
   #  end
   #
 
+  mdc.subspec "schemes" do |scheme_spec|
```

Comment:
Why not have 

MaterialComponents/Color
MaterialComponents/Motion
MaterialComponents/Shape
MaterialComponents/Typography

Why do we want a /scheme/* ?

Reply:
They aren't and shouldn't be considered as components.